go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -115,36 +115,135 @@ func TestSingleQuery(t *testing.T) {
 
 // Convenience test for debugging a single query. Unskip and set to the desired query.
 func TestSingleScript(t *testing.T) {
-	t.Skip()
+	// t.Skip()
```

Comment:
Might as well keep this skipped for check-in

```suggestion
	t.Skip()
```